testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveMetastoreClientFactory.java

```diff
@@ -50,8 +49,7 @@ public ThriftMetastoreClient createMetastoreClient()
             throws TException
     {
         URI metastore = URI.create("thrift://" + metastoreHost + ":" + metastorePort);
-        return thriftMetastoreClientFactory.create(
-                HostAndPort.fromParts(metastore.getHost(), metastore.getPort()),
+        return thriftMetastoreClientFactory.create(metastore,
```

Comment:
Either specify the parameters each on a  new line or (in this specific case)  wrap them in one line

```suggestion
return thriftMetastoreClientFactory.create(metastore, Optional.empty());
```